net_util/src/queue_pair.rs

```diff
@@ -98,6 +98,7 @@ impl TxVirtio {
                     /* EAGAIN */
                     if e.kind() == std::io::ErrorKind::WouldBlock {
                         warn!("net: tx: (recoverable) failed writing to tap: {}", e);
+                        queue.go_to_previous_position();
```

Comment:
I thought we already rewind the queue (line 96) for all the "unsuccessfully" `libc::writev`. Is it intentionally to have the second rewind for EAGAIN?

Reply:
Well spotted! I missed this because it's not quite the same pattern as the other code where the rewind happens immediately before the `break`.